<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>451</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Monospace</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Genético aplicado a problemas PL</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="PLGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Problema PL</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>411</width>
       <height>122</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="zLabel">
        <property name="text">
         <string>Z =</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="addRestPushButton">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="zLineEdit">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>a + b</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="saLabel">
        <property name="text">
         <string>Sujeto a</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="geneticGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>431</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Algoritmo Genético</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>411</width>
       <height>137</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="generacionLabel">
        <property name="text">
         <string>Generaciones</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="generacionLineEdit">
        <property name="text">
         <string>50</string>
        </property>
        <property name="placeholderText">
         <string>Número de iteraciones</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="poblacionLabel">
        <property name="text">
         <string>Población</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="poblacionLineEdit">
        <property name="text">
         <string>100</string>
        </property>
        <property name="placeholderText">
         <string>Tamaño de la población</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="presicionLabel">
        <property name="text">
         <string>Precisión</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="precisionLineEdit">
        <property name="text">
         <string>5</string>
        </property>
        <property name="placeholderText">
         <string>Bits de precisión</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="maxRadioButton">
        <property name="text">
         <string>Maximizar</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QRadioButton" name="minRadioButton">
        <property name="text">
         <string>Minimizar</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="aptitudLabel">
        <property name="text">
         <string>Aptitud (Z)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="runPushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>380</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
